test/e2e/util.go

```diff
@@ -652,9 +658,9 @@ func waitForPersistentVolumePhase(phase api.PersistentVolumePhase, c *client.Cli
 	return fmt.Errorf("PersistentVolume %s not in phase %s within %v", pvName, phase, timeout)
 }
 
-// createTestingNS should be used by every test, note that we append a common prefix to the provided test name.
+// CreateTestingNS should be used by every test, note that we append a common prefix to the provided test name.
 // Please see NewFramework instead of using this directly.
-func createTestingNS(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error) {
+func CreateTestingNS(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error) {
```

Comment:
well, what I've been wondering is whether we should always call `util.createTestingNS` from `framework.CreateNamespace`, but if the `testContext` ns func is set, call that first?

it makes it a little clearer why you'd set the `testContext` one in the first place, I think.